src/modify.js

```diff
@@ -92,20 +94,43 @@ function rewriteAllFields(schema, configCallback, context) {
   });
 }
 
+function reorderFields(schema, configOrder) {
+  if (!configOrder) return { warnings: null };
+
+  const warnings = [];
+  const originalOrder = schema['x-jsf-order'];
```

Comment:
Is `x-jsf-order` a required property of a json-schema? If not we might need a fallback here.

Reply:
good point, added fallback and a test!

Reply:
🙇 🙇 🙇